postman/postman_collection.json

```diff
@@ -34,15 +36,15 @@
 				"header": [],
 				"body": {
 					"mode": "raw",
-					"raw": "{\n    \"name\": \"NoDe tow\",\n    \"description\": \"I'm a little node\"\n}",
+					"raw": "{\n    \"name\": \"Node two\",\n    \"description\": \"Something\"\n}",
 					"options": {
 						"raw": {
 							"language": "json"
 						}
 					}
 				},
 				"url": {
-					"raw": "http://127.0.0.1:5151/node",
+					"raw": "http://127.0.0.1:5151/node/create",
```

Comment:
As it was discussed before, APIs url should not be in this form `/node/create` etc, please fix the postman collection or revert the changes.